source/gloperate-text/source/FontFace.cpp

```diff
@@ -111,12 +111,12 @@ void FontFace::setGlyphTexturePadding(const glm::vec4 & padding)
 
 globjects::Texture * FontFace::glyphTexture() const
 {
-    return m_glyphTexture;
+    return m_glyphTexture.get();
 }
 
 void FontFace::setGlyphTexture(globjects::Texture * texture)
 {
-    m_glyphTexture = texture;
+    m_glyphTexture.reset(texture);

```

Comment:
use `std::unique_ptr<globjects::Texture> &&` as parameter and then `m_glyphTexture = std::move(texture)`